cpp/tests/community/mg_louvain_helper.cu

```diff
@@ -71,7 +71,7 @@ bool compare_renumbered_vectors(raft::handle_t const &handle,
                                     v1.end(),
                                     vertex_t{0});
 
-  rmm::device_uvector<size_t> map(max, size_t{0});
+  rmm::device_uvector<size_t> map(max, cudaStream_t{nullptr});
```

Comment:
Should we just use handle.get_stream() here?

Reply:
Yes, we SHOULD.